### ISSUE TITLE
One-Ways are in the wrong direction (sometimes)

### ISSUE THREAD

**Issue by Bissel** [mapbox/mapbox-unity-sdk#1723]
_Note: If this is a bug or support ticket, please provide the following information:_ **Bug**
* Unity version: 2020.1.7f1
  * Scripting Runtime Version: 
  * Scripting Backend: Mono
  * Api Compatibility Level: .NET Standard 2.0
* Mapbox SDK version: 2.0.1
* The platform you're building to: Windows
* A description of what you're trying to do: I am trying to set waypoints to the Points in the `feature.Points` list, in the documentation it's stated, that the points are ordered in the direction of the one way road.
* Steps to recreate the bug if appropriate: 
  * Map settings are the folliowing
    * Latitude Longitude: `53.07694, 8.80889`
    * Zoom: `16`
    * Extent Options: Range Around Transform 1 1
    * Placement Options: At Location Center
    * Scaling Options: World Scale
    * MapLayers DataSource: Mapbox Streets V8 with Building Ids
  * Create a custom MeshModifier (and use it in the map on data layer `road`)
  * find oneway roads (`feature.Properties["oneway"]`)
  * road-name examples: (`feature.Properties["name"]`) "Violenstraße" "Domshof"
    * only some of the roads are in the wrong direction

* Links to your logs are also helpful:

**Comment by brnkhy** [comment]
@Bissel I wonder if the data is wrong for those locations, have you checked it with mapbox studio? I mean to see if it's the data or something on the unity side.

**Comment by Bissel** [comment]
@brnkhy the one way roads on the map (texture) are displayed correctly and the directions api works also correct. 
Is there a hook function so I can check the raw data that is provided by mapbox?

**Comment by brnkhy** [comment]
@Bissel so if I understand it correct, you are saying `one way` information in road layer is wrong while images on mapbox studio and directions from directions api are correct? That's interesting. I wonder if they are using a more up-to-date source.
Easiest way to check features are mesh modifiers. They receive the feature information so you can check what data looks like easily.

**Comment by Bissel** [comment]
@brnkhy yes thats what I'm trying to say. I also checked OSM and the direction of the mentiond roads are correct there.
I am using a MeshModifier as described in the Issue but some of the Point-Lists in `feature.Points` are not Pointing in the correct direction.
